task3-2.js

```diff
@@ -0,0 +1,33 @@
+"use strict"
+
+let sarah = {
```

Comment:
don't use global variables. For this task please use input params directly for the desctucturization